runner_scripts/sanitize_jfrog_artifactory.py

```diff
@@ -0,0 +1,74 @@
+import pyartifactory
+import sys
+
+ARTIFACTORY_BASE_URL = "https://cura.jfrog.io/artifactory"
+
+if len(sys.argv) != 3:
+    print("Usage: python your_script.py USERNAME PASSWORD")
+    exit()
+
+USERNAME = sys.argv[1]
+PASSWORD = sys.argv[2]
+
+def initialize_artifactory():
+    return pyartifactory.Artifactory(url=ARTIFACTORY_BASE_URL, auth=(USERNAME, PASSWORD))
+
+ARTIFACT_PATHS = {"cura-conan-dev-local/ultimaker/curaengine"          : True,
+                  "cura-conan-dev-local/ultimaker/cura"                : True,
+                  "cura-conan-dev-local/ultimaker/fdm_printer"         : True,
+                  "cura-conan-dev-local/ultimaker/uranium"             : True,
+                  "cura-conan-dev-local/_/curaengine"                  : True,
+                  "cura-conan-dev-local/_/cura"                        : True,
+                  "cura-conan-dev-local/_/fdm_printer"                 : True,
+                  "cura-conan-dev-local/_/uranium"                     : True,
+                  "cura-conan-cci-remote-cache/ultimaker/curaengine"   : True,
+                  "cura-conan-cci-remote-cache/ultimaker/cura"         : True,
+                  "cura-conan-cci-remote-cache/ultimaker/fdm_printer"  : True,
+                  "cura-conan-cci-remote-cache/ultimaker/uranium"      : True,
+                  "cura-conan-cci-remote-cache/_/curaengine"           : True,
+                  "cura-conan-cci-remote-cache/_/cura"                 : True,
+                  "cura-conan-cci-remote-cache/_/fdm_printer"          : True,
+                  "cura-conan-cci-remote-cache/_/uranium"              : True,
+                  "cura_conan-cci-remote-cache"	                       : False}
+
+
+def list_artifacts(artifactory_client, artifact_path, depth):
+    try:
+        return artifactory_client.artifacts.list(f"{artifact_path}/", depth).files
+    except pyartifactory.exception.ArtifactoryError as e:
+        # artifact_path was not found in the repository, so we return empty dict
+        # print(f"Repository {artifact_path} does not exist")
+        return {}
+
+
+def delete_artifact(artifactory_client, artifact_path):
+    artifactory_client.artifacts.delete(artifact_path)
+
+
+def artifact_modified_by_anonymous(artifactory_client, artifact_path):
+    return str(artifactory_client.artifacts.info(artifact_path).createdBy) == "anonymous"
+
+
+def process_artifact(artifactory_client, artifact_path, sequence_nr, file, depth):
```

Comment:
`file` argument is not used

Reply:
removed b04a655c9a5a58cd15ca0abe60594139ec82ef8a